content/getting-started/community.md

```diff
@@ -5,11 +5,30 @@ prev: "/getting-started/solutions/"
 
 # Community
 
-<!--
-TODO Kata discussion page, posts, spoilers, post labels
-TODO How to communicate effectively
-TODO Where to ask for help
-TODO Community rules
-TODO Link to Zulip when ready
-TODO Link to Forum when ready
--->
+Codewars gathers thousands of users of many skills and levels of experience. Their knowledge is invaluable resource and when used correctly, your training can be easier, and extremely effective.
+
+## Kata Discourse
+
+If you want to discuss some particular kata, you can use its discourse page to do so. It's a good place to ask for explanation, search for help with solution, add your opinion on it, or report problems. You can label your post with one of possible labels:
+
+- **Question** - if description is not clear, you need help, got stuck, or you cannot get your solution to work,
+- **Suggestion** - if you have an idea how author could change the kata, its description, or tests, to make it better,
+- **Issue** - if you think the kata is broken, and author needs to fix it in some way.
+
```

Comment:
```suggestion

See this [detailed documentation (TODO: direct link to Labels reference)]() about the different kinds of labels and their use.

```

Reply:
Since posting bogus issues is one of the most common beginner mistakes, I think it will be good to have labels briefly explained here (with a link to actual, full definition)

Reply:
yes, my suggestion is just the addition of that link (with emphasis about its existence).